packages/liferay-npm-scripts/src/scripts/webpack.js

```diff
@@ -46,15 +46,19 @@ module.exports = function(...args) {
 	}
 };
 
+function escapeLiteralString(str) {
+	return str.replace(/\\/g, '\\\\');
```

Comment:
See my comment above... should we replace `\` rather than `\\`?

Reply:
Ugh, that's what this is doing, right? I always get confused here 😂

Reply:
Yes. Backslash escaping is a pain :-S.